### DISCUSSION TITLE
Display total on first level cell's tooltip

### DISCUSSION THREAD

**Discussion by Chatcola** [marcusolsson/grafana-treemap-panel#37] [Ideas]
Thanks for this amazing plugin !
The first level cell's tooltip just display the label and the second level cell's tooltip display label: value by default as linked on the capture.
I couldn't find a way to add the total beside renaming the value with the correct syntax which isn't preferable, thanks by advance I hope you had a nice holyday!
![treemap_tooltip](https://user-images.githubusercontent.com/25823356/147395901-b1303319-8c02-4b17-bf9b-01db516f20e7.png)

**Comment by marcusolsson** [comment]
I think this is a good idea! I'm not sure when I'll have time to work on this, but I'd be happy to review a pull request!
